app/src/main/res/layout/toolbar.xml

```diff
@@ -67,16 +67,35 @@
         android:layout_gravity="center_vertical"
         android:background="?android:attr/selectableItemBackgroundBorderless"/>
 
-    <ImageButton
-        android:id="@+id/menu"
-        android:layout_width="48dp"
-        android:layout_height="48dp"
-        android:layout_gravity="center_vertical"
-        android:layout_marginBottom="8dp"
-        android:layout_marginTop="8dp"
-        android:background="?android:attr/selectableItemBackgroundBorderless"
-        android:padding="4dp"
-        android:contentDescription="@string/content_description_menu"
-        android:src="@drawable/ic_menu" />
+    <FrameLayout
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:paddingTop="4dp"
+        android:paddingEnd="4dp">
```

Comment:
This padding moved the menu button slightly to the left. It's almost not noticeable. :)

Maybe you can move the badge view with "margin" into position instead?